test/jdk/javax/swing/GraphicsConfigNotifier/StalePreferredSize.java

```diff
@@ -182,10 +185,6 @@ static void checkComponent(Callable<JComponent> creator) throws Exception {
                 int y = frame.getY() + 200;
                 PopupFactory factory = PopupFactory.getSharedInstance();
                 popup = factory.getPopup(frame, component, x, y);
-                if (component instanceof JMenuItem) {
-                    // TODO JDK-8244400
-                    MenuItemLayoutHelper.clearUsedParentClientProperties((JMenuItem)component);
```

Comment:
This call was the only reason to have " * @modules java.desktop/sun.swing" at the header of the test.

Reply:
ok..removed..